src/SamplesApp/SamplesApp.Droid/Resources/xml/network_security_config.xml

```diff
@@ -0,0 +1,5 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+  <base-config cleartextTrafficPermitted="true" >
```

Comment:
I don't think we should be shipping with insecure defaults. How about we change the UI test to use https:// instead of http:// and then burn this file.

Reply:
We're not shipping this app. The point of this modification is to restore the many samples that use this kind of urls, and there are many that are not tested properly after the upgrade to Android 9.0.

I'll move this change to another PR, as there a about 270 Urls that need to be validated to determine if they do work in https.